### ISSUE TITLE
Add example with variable susceptibility

### ISSUE THREAD

**Issue by joelmiller** [springer-math/Mathematics-of-Epidemics-on-Networks#32]
I'd like to add an example with variable susceptibility for either or both SIR and SIS.  If you've got a specific case you're interested in, let me know and I'll try to work it out.

**Comment by paigemiller** [comment]
I would be interested in an example with variable susceptibility for the SIR model but also for the SIRS model if possible. Specifically for a network with male and female nodes, I would like susceptibility to depend on node sex such that male susceptibility is twice female susceptibility.

**Comment by joelmiller** [comment]
Hmm,  SIR with variable susceptibility is straightforward with the current code.

SIRS with variable susceptibility isn't yet.  But it should be.  The 'simple_contagion' code allows you to use an edge weight in the contact network, but I should have set it up to instead allow you to pass in a function which would handle it.  It should be a straightforward modification, but it'll be a few days at least before I have time to do it.  Send me a reminder [and repeat reminders every couple days], if I haven't gotten around to this when you need it.

Basically what's needed is that the `simple_contagion` code will have to allow the input transition graphs to also have a function associated with the edge representing a transition.  The function will take the two nodes and the graph (which could have node attributes) and any additional parameters and produce a rate.

**Comment by joelmiller** [comment]
I believe this is addressed as of version 1.0.8rc1
